### PR TITLE
Create prefix.js

### PR DESCRIPTION
**Pull request by gc-au**
This is a shortcut for setting prefix with `+conf set prefix`. more user-friendly and will show up in +help.

### CODE REVIEW

**Comment by tech6hutch** [comment]
If there's going to be a command called `prefix`, I think using it without args should show the current prefix, like using the mention by itself does.

**Comment by kyranet** [comment]
Please lint your code with [klasa-lint](https://github.com/dirigeants/klasa-lint/blob/master/.eslintrc.json).

File: commands/Configuration/prefix.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			permLevel: 6,
+			runIn: ['text'],
+			cooldown: 5,
+			description: 'Change the command prefix the bot uses in your server.',
+			usage: '[prefix:str{1,4}]'
```
Comment (tech6hutch) [inline commands/Configuration/prefix.js]: Can't prefixes be up to 10 chars?

Comment (kyranet) [reply commands/Configuration/prefix.js]: Yes, they do.

**Pull request merged**